backend/core/models.py

```diff
@@ -471,6 +471,7 @@ class Team(CommonInfo):
     name = models.CharField(max_length=255)
     is_public = models.BooleanField(default=False)
     recipes = GenericRelation("Recipe", related_query_name="owner_team")
+    # deprecated
     ical_id = models.TextField(
```

Comment:
you could [set the column name](https://docs.djangoproject.com/en/3.0/ref/models/fields/#django.db.models.Field.db_column) and rename the field to `deprecated_ical_id`.

```python
deprecated_ical_id = models.TextField(
    db_column="ical_id",
    default=get_random_ical_id
    help_text="Secret key used to prevent unauthorized access to schedule calendar.",
)
```